Source/WebCore/accessibility/AccessibilityRenderObject.cpp

```diff
@@ -2589,7 +2589,19 @@ void AccessibilityRenderObject::addChildren()
 
     if (node && !(element && element->isPseudoElement()) && cache) {
         // If we have a DOM node, use the DOM to find accessible children.
-        for (Ref child : composedTreeChildren(*node)) {
+#if PLATFORM(IOS_FAMILY)
```

Comment:
Maybe it'd be better to use the InlineContextCapacity of 0 on all platforms? Even though macOS has a larger stack size, it's probably still a good idea to avoid putting so much on the stack. If it turns out there are consequences of this change, I'd rather we experience them on all platforms.

Reply:
Yeah, makes sense! Made this change in the latest commit.